### DISCUSSION TITLE
Fly again, 2024

### DISCUSSION THREAD

**Discussion by xiaodong2008** [fastjs-team/core#59] [Announcements]
Hi, I am dy-xiaodong2022. Maintainer of this project, this project is on the schedule. Thanks you for all.

### 致国内开发者
在这段时间里经历了太多，我也认识到了自己的不足，希望在这次重构之后可以给大家带来一个~~优秀~~普通的开源项目

**Comment by liduchuan** [comment]
你很棒哦！！！不过考虑到国内的教育环境问题，建议怎么开心怎么来 😁

**Comment by Simon-He95** [comment]
加油⛽️～
